### PR TITLE
Internal: Display a different 'Getting Started' section for free users in the Home screen [ED-14098]

### PR DESCRIPTION
**Pull request by elchugreeva**

### CODE REVIEW

File: modules/home/transformations/filter-get-started-by-license.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+namespace elementor\modules\home\transformations;
+
+use Elementor\Modules\Home\Transformations\Base\Transformations_Abstract;
+use Elementor\Utils;
+
```
Comment (louiswol94) [inline modules/home/transformations/filter-get-started-by-license.php:6]:
```suggestion
```

File: tests/phpunit/elementor/modules/home/transformations/test-filter-get-started-by-license.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+		// Arrange
+		$original_data = $this->mock_home_screen_data();
+
+		$transformation = new Filter_Get_Started_By_License( [] );
+		$transformation->has_pro = true;
+
+		// Act
+		$transformed_data = $transformation->transform( $original_data );
+		$expected_data = $this->mock_home_screen_data_transformed_pro();
+		var_dump($transformed_data);
```
Comment (louiswol94) [inline tests/phpunit/elementor/modules/home/transformations/test-filter-get-started-by-license.php:33]:
```suggestion

```

File: tests/phpunit/elementor/modules/home/transformations/test-filter-get-started-by-license.php [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+		// Arrange
+		$original_data = $this->mock_home_screen_data();
+
+		$transformation = new Filter_Get_Started_By_License( [] );
+		$transformation->has_pro = true;
+
+		// Act
+		$transformed_data = $transformation->transform( $original_data );
+		$expected_data = $this->mock_home_screen_data_transformed_pro();
+		// Assert
```
Comment (louiswol94) [inline tests/phpunit/elementor/modules/home/transformations/test-filter-get-started-by-license.php:33]:
```suggestion
		$expected_data = $this->mock_home_screen_data_transformed_pro();

		// Assert
```

File: modules/home/transformations/filter-get-started-by-license.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	public function __construct( $args ) {
+		parent::__construct( $args );
+
+		$this->has_pro = Utils::has_pro();
+	}
+
+	private function valid_item( $item ) {
+		$has_pro_json_not_free = $this->has_pro && 'pro' === $item['license'][0];
+		$is_not_pro_json_not_pro = ! $this->has_pro && 'free' === $item['license'][0];
+		return $has_pro_json_not_free || $is_not_pro_json_not_pro;
```
Comment (louiswol94) [inline modules/home/transformations/filter-get-started-by-license.php:23]:
```suggestion
		$is_not_pro_json_not_pro = ! $this->has_pro && 'free' === $item['license'][0];

		return $has_pro_json_not_free || $is_not_pro_json_not_pro;
```

**Review by louiswol94 — DISMISSED** [review]

File: modules/home/assets/js/components/home-screen.js [hunk 5]
```diff
@@ -1,5 +1,4 @@
 import { Container, Box, Stack } from '@elementor/ui';
```
Comment (hein-obox) [inline modules/home/assets/js/components/home-screen.js:1]: Can we keep this empty line?

File: modules/home/transformations/filter-get-started-by-license.php [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<?php
+namespace elementor\modules\home\transformations;
```
Comment (louiswol94) [inline modules/home/transformations/filter-get-started-by-license.php:2]:
```suggestion
namespace Elementor\Modules\Home\Transformations;
```

**Review by louiswol94 — DISMISSED** [review]

File: modules/home/assets/js/components/home-screen.js [hunk 7]
```diff
@@ -1,5 +1,4 @@
...
-
 import TopSection from './top-section';
```
Comment (hein-obox) [inline modules/home/assets/js/components/home-screen.js:2]: I don't think that the current PR is working.

I think that this line needs to be updated first: 
`<GetStarted getStartedData={ props.homeScreenData.get_started[ 0 ] } />`

Also I believe we need to make this conditional.

**Review by hein-obox — APPROVED** [review]

**Pull request merged**